frontends/web/src/i18n/i18n.js

```diff
@@ -100,8 +100,8 @@ i18n.on('languageChanged', (lng) => {
     if (!match) {
       // There are too many combinations. So, we compare only the main
       // language tag.
-      const lngLang = lng.replace('_', '-').split('-')[0];
-      const localeLang = nativeLocale.replace('_', '-').split('-')[0];
+      const lngLang = localeMainLanguage(lng);
+      const localeLang = localeMainLanguage(nativeLocale);
```

Comment:
deserves its own commit. i'd always try to think about if a commit needed to be reverted, you'd only want to revert one logical change, not many. or if you bisect a bug, you should land on a commit with one change, not many.

Reply:
Good point! Done